enqueue/amqp-bunny/0.7/post-install.txt

```diff
@@ -0,0 +1,9 @@
+<bg=yellow;fg=black>                             </>
+<bg=yellow;fg=black> Using amqp-bunny in Enqueue </>
+<bg=yellow;fg=black>                             </>
+
+  * Configure <comment>ENQUEUE_DSN=amqp+bunny://user:pass@example.com:10000/%2f</comment>
```

Comment:
`amqp+bunny:` adds default options for host, port, user and password. The most commonly used. So I am not sure we should provide more details other than that. 

If you still want to make it more descriptive I'd suggest 

```
ENQUEUE_DSN=amqp+bunny://guest:guest@localhost:5672/%2f
```

Reply:
Ah, ok cool. I've changed to use the more descriptive suggestion then. This also solves the problem with the 0.7 version requiring the `//` - the example now shows WITH `//`, so this recipe is now compat with all versions.